easybuild/easyconfigs/s/Subread/Subread-2.0.0-GCC-10.2.0-duplicate-symbol.patch

```diff
@@ -0,0 +1,11 @@
+--- subread-2.0.0-source.original/src/gene-algorithms.h	2019-09-04 00:22:49.000000000 -0400
```

Comment:
@justbennet could you add a comment to the patch explaining the purpose in a bit more detail please?

Reply:
I added this commet,
```
#  Changing the declaration of replica_index to extern prevents
#  ld.gold from erroring with a 'duplicate symbol found' message.
```
Good?

Reply:
lgtm